content/blog/ai-meets-tf-prompt-strategies-for-test-generation.md

```
@@ -0,0 +1,199 @@
+> TLDR: We crafted an LLM prompt to generate Terraform tests. It's [here](#) in our [shared-prompts](https://github.com/masterpointio/shared-prompts) Github repo. If you’re looking for the list of things that worked for us, jump to the [Takeaways for Durable Prompts](#takeaways-for-durable-prompts) section. If you want to learn how we got there, read on.
+
+## Table of Contents
+- [Why AI for IaC?](#why-ai-for-iac)
+- [Our AI Toolkit for IaC](#our-ai-toolkit-for-iac)
+  - [AI-Enhanced IDE](#ai-enhanced-ide)
+  - [Terminal-Based AI Coding](#terminal-based-ai-coding)
+  - [We’re still experimenting with MCPs for Code Gen](#were-still-experimenting-with-mcps-for-code-gen)
+- [Our exploration in Writing Terraform Tests with AI](#our-exploration-in-writing-terraform-tests-with-ai)
+  - [v0 – Cursor Auto Model](#v0--cursor-auto-model)
+  - [v1 – Claude Code using Sonnet-4](#v1--claude-code-using-sonnet-4)
+  - [v2 – Cursor + Sonnet-4 + Refined Prompt](#v2--cursor--sonnet-4--refined-prompt)
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid

(MD051, link-fragments)